simplified-opds-core/src/main/java/org/nypl/simplified/opds/core/OPDSJSONSerializer.java

```diff
@@ -67,6 +69,21 @@ public ObjectNode serializePreviewAcquisition(
       return node;
   }
 
+  private String getFullMimeTypeInfo(MIMEType mimeType) {
```

Comment:
I would name this `serializeContentType`, to be similar to the other functions in this file that do similar things.